targets/TARGET_STM/can_api.c

```diff
@@ -21,29 +21,34 @@
 #include "pinmap.h"
 #include "PeripheralPins.h"
 #include "mbed_error.h"
+#include "can_device.h" // Specific to STM32 serie
 #include <math.h>
 #include <string.h>
 
-#define CAN_NUM    2
 static CAN_HandleTypeDef CanHandle;
 static uint32_t can_irq_ids[CAN_NUM] = {0};
 static can_irq_handler irq_handler;
 
-void can_init(can_t *obj, PinName rd, PinName td)
-{
-    uint32_t filter_number;
+void can_init(can_t *obj, PinName rd, PinName td) {
```

Comment:
I am confused now. I thought the correct coding style was to have the { at the end of the function name ?